zdm/survey.py

```diff
@@ -592,15 +713,13 @@ def load_survey(survey_name:str, state:parameters.State, dmvals:np.ndarray,
     # Do it
     srvy=Survey()
     srvy.name = survey_name
-    srvy.process_survey_file(os.path.join(sdir, dfile), NFRB=NFRB, iFRB=iFRB)
+    path=os.path.join('/Users/cjames/CRAFT/FRB_library/Git/H0paper/zdm/data/Surveys/',dfile)
```

Comment:
no absolute paths to `cjames`!  :)

Reply:
Oops, I know why this is there. It's because you implemented your darn if (name=thing) then (get it from ...) notation, and I needed to looks somewhere else! Forgot this temporary cheat was in there. I'll undo this. Can we undo this from the web page?

Reply:
Unwound this now